picky-krb/Cargo.toml

```diff
@@ -28,3 +29,10 @@ aes = "0.8.1"
 des = "0.8.1"
 cbc = "0.1.2"
 rand = "0.8.5"
+
+uuid = { version = "1.1.2", default-features = false, features = ["serde"] }
+oid = "0.2.1"
+num-derive = "0.3.3"
+num-traits = "0.2.15"
+num-bigint = { version = "0.4.3", features = ["rand"] }
+bincode = "1.3.3"
```

Comment:
What is motivating `bincode` addition? I don’t see any usage 🤔

Reply:
I've planned to use it for the NEGOEX data types parsing/encoding but it turned out not to be convenient. I will remove it